src/_base/harness/attributes/docker-base.yml

```diff
@@ -89,6 +92,8 @@ attributes:
         nginx:
           environment:
             FPM_HOST: localhost
+          ingress:
+            annotations: {}
```

Comment:
Does this still get picked up by `.Values.docker.services.nginx.ingress.annotations` in `src/_base/helm/app/templates/application/webapp/ingress.yaml` and `src/spryker/helm/app/templates/application/webapp/ingress.yaml`?

Reply:
yes, it's filtered out of workspace attribute merge of services.*, but pipeline.services.* is left unaltered